client/src/pages/ResetPassword/ForgetPassword/ForgetPassword.tsx

```diff
@@ -0,0 +1,47 @@
+import { Grid, CssBaseline, Paper, Box, Typography } from '@material-ui/core';
+import { FormikHelpers } from 'formik';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import sendResetEmailAPI from '../../../helpers/APICalls/resetPassword';
+import EmailForm from './EmailForm/EmailForm';
+import { useHistory } from 'react-router-dom';
+import useStyles from './useStyles';
+
+export default function ForgetPassword(): JSX.Element {
+  const classes = useStyles();
+  const history = useHistory();
+  const { updateSnackBarMessage } = useSnackBar();
+  const handleSubmit = ({ email }: { email: string }, { setSubmitting }: FormikHelpers<{ email: string }>) => {
+    sendResetEmailAPI(email).then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else {
+        updateSnackBarMessage(data.success.message);
+        setTimeout(() => {
+          history.push(`/email/reset-password/${data.success.token}`);
+        }, 2000);
+      }
+    });
+    setSubmitting(false);
+  };
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid item xs={12} elevation={6} component={Paper} square>
+        <Box display="flex" alignItems="center" justifyContent="center" flexDirection="columu" paddingTop="23px">
+          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
+            <Grid container>
+              <Grid item xs component={Paper} elevation={3} square className={classes.authCard}>
```

Comment:
Won't this Paper component by overridden by the parent Paper component, maybe exclude it?

Reply:
tried to remove the paper component, and it seems will remove the paper effects, so I think it can not be excluded